pimd/pim_nb_config.c

```diff
@@ -1573,7 +1533,7 @@ int lib_interface_pim_address_family_destroy(struct nb_cb_destroy_args *args)
 		if (!pim_ifp)
 			return NB_OK;
 
-		if (!pim_cmd_interface_delete(ifp)) {
+		if (!pim_pim_interface_delete(ifp)) {
```

Comment:
This function never returns something except 1, does it makes sense for this check?

Reply:
Removed this check.